src/Illuminate/View/Compilers/BladeCompiler.php

```diff
@@ -277,7 +277,7 @@ protected function compileRawEchos($value)
 		{
 			$whitespace = empty($matches[3]) ? '' : $matches[3].$matches[3];
 
-			return $matches[1] ? substr($matches[0], 1) : '<?php echo '.$this->compileEchoDefaults($matches[2]).'; ?>'.$whitespace;
+			return $matches[1] ? substr($matches[0], 1) : '<?= '.$this->compileEchoDefaults($matches[2]).'; ?>'.$whitespace;
```

Comment:
@crynobone PHP5.4+ has `<?=` short tag enabled by default. And Laravel requires 5.4+, so why not?